fix(ChartModal): :bug: Display descriptive error message if selected correlation fields do not contain all numeric values

person-al opened the pull request:
Added a check to ensure that if user selected a field for the correlation chart modal and the field did not pass the "isQuant" check, a descriptive error is displayed

Closes issue #13

SkepticMystic commented on src/Components/Chart.svelte:
```diff
@@ -65,7 +66,15 @@
+		if (isValidSelection(f1, f2)) {
+			const xValid = isQuant(xs)
+			const yValid = isQuant(ys)
+			errorMessage = !xValid ? "First parameter has non-numeric values" : errorMessage
+			errorMessage = !yValid ? "Second parameter has non-numeric values" : errorMessage
```
It could be that both parameters have non-numeric values. Perhaps append to the string instead of overwriting it to show that both have errors

SkepticMystic commented on src/analyses.ts:
```diff
@@ -7,14 +7,14 @@
+
 	if (
-		xs.length <= 1 ||
-		ys.length <= 1 ||
-		skipTest ||
-		!isQuant(xs) ||
-		!isQuant(ys)
+		!skipQuantCheck &&
+		(!isQuant(xs) || !isQuant(ys))
```
Nice! Thanks for getting this right

Commit:
fix(ChartModal): :bug: Display descriptive error message if selected correlation fields do not contain all numeric values

    Added a check to ensure that if user selected a field for the correlation chart modal and the field did not pass the isQuant check, a descriptive error is displayed

    Closes issue #13

person-al commented:
Closed in favor of: https://github.com/SkepticMystic/data-analysis/pull/23
